src/I18n/Date.php

```diff
@@ -325,7 +325,3 @@ public function __toString(): string
         return (string)$this->i18nFormat();
     }
 }
-
-// phpcs:disable
-class_alias(Date::class, 'Cake\I18n\FrozenDate');
```

Comment:
Why are we removing these?

Reply:
I thought those are removed in 5.x
Apparently, they are not. I will see why the validation script reports them as missing and reopen the PR if fixed.